package.json

```diff
@@ -38,9 +39,9 @@
     "eslint-plugin-sql-template": "^2.0.0"
   },
   "devDependencies": {
-    "@uphold/github-changelog-generator": "^1.0.1",
-    "eslint": "^7.18.0",
-    "mocha": "^7.1.1",
+    "@uphold/github-changelog-generator": "^3.0.0",
```

Comment:
This bump changes the way the generator works, I believe it needs to change to this:

```bash
echo "$(github-changelog-generator --future-release=v$npm_config_future_release)\n$(tail -n +2 CHANGELOG.md)" > CHANGELOG.md
```

Reply:
https://github.com/uphold/uphold-scripts/pull/43/files#diff-1dc804c678844f463aff4d9d0214e418b6b1f8952b8f819171698b328c589af7R3

yep

Reply:
Maybe do it in a different PR?